### ISSUE TITLE
C# AlprNetGuiTest Fails on Recognize

### ISSUE THREAD

**Issue by fdncred** [openalpr/openalpr#625]
I went the hard way and compiled Leptonica, OpenCv, & Tesseract in an attempt to get OpenALPR compiled. **Surprise! I got it all compiled just fine.** 

I want to use OpenALPR in a C# application however the alprnet bindings reference [libopenalprc.dll](https://github.com/openalpr/openalpr/blob/master/src/bindings/csharp/alprnet/Alpr.cs#L305) which doesn't exist. 

After looking at the functions it was trying to call I think it should be using [DllImport("openalpr.dll")] since that's where functions like openalpr_init() exist.  So, I changed that in Alpr.cs and **I was able to successfully initialize the library via openalpr_init()** using the sample AlprNetGuiTest which calls aplr.Initialize(). That finally worked when I got all the dependencies in the debug folder. However, the next library function is alpr.Recognize() which calls **openalpr_recognize_rawimage()** or **openalpr_recognize_encodedimage()**. **Both of these functions fail** in the dotNet version with _"AccessViolationException: 'Attempted to read or write protected memory...'"_.

Any tips on how to debug this?

Thanks,
Darren

**Comment by fdncred** [comment]
One problem fixed. Another found.
My problem was with OpenCv. I figured this out by trying to debug the alpr.exe program. imread() wouldn't load the file. So, several hours later after recompiling, i'm initializing the engine and loading the file just fine. 

Now my problem is with Tesseract. LTRResultIterator::WordFontAttributes() is throwing an exception _PAGE_RES:IT::row returned nullptr_. I tested out the tesseract.exe and it works fine. So I'm not sure why function is failing.

Any tips?
